app/models/pacbio/run.rb

```diff
@@ -46,7 +46,12 @@ class Run < ApplicationRecord
 
     # if comments are nil this blows up so add try.
     def comments
-      super || wells.try(:collect, &:summary).try(:join, ':')
+      return super if super.present?
+
+      wells.collect do |well|
+        "#{well.used_aliquots.first.source.tube.barcode} " \
+          "#{well.smrt_link_options['library_concentration']}pM"
```

Comment:
You can access library_concentration directly i.e `well.library_concentration`,

smrt_link_options are a store which provides accessors.